src/Game/Managers/MacroManager.cs

```diff
@@ -1066,6 +1066,7 @@ private int Process(MacroObject macro)
                  case MacroType.CloseAllHealthBars:
 
                     var healthBarGumps = Engine.UI.Gumps.OfType<HealthBarGump>();
```

Comment:
Could tidy this block up by just having this: 
```
var healthBarGumps = Engine.UI.Gumps.Where(g => g is HealthBarGump \|\| g is HealthBarGumpCustom);
                    foreach (var healthbar in healthBarGumps)
                        {
                            if (Engine.UI.AnchorManager[healthbar] == null && (healthbar.LocalSerial != World.Player))
                            {
                                healthbar.Dispose();
                            }
                        }
                    break;
```

Reply:
I originally did this but decided to use a different anchor name for customhealthbars because last pr someone said mentioned anchor group name thus you have 2 groups

Reply:
Your replay makes no sense. The code above is doing exactly what the code in place is doing, just less code 🍡

Reply:
I am all for making the code cleaner, however my response above was wrong as I was on mobile and not sitting here looking at the code itself. Your way of handling the gumps via Gumps.Where throws an argument because it's trying to convert your control 'g is a or b' to an AnchorableGump. This is why I had to use Gumps.OfType for each healthbartype. If you can find another solution to this by using less code let me know, I learn as I go :)

Reply:
Oh, try 
`
var healthBarGumps = Engine.UI.Gumps.OfType<AnchorableGump>().Where(g => g is HealthBarGump || g is HealthBarGumpCustom);
`

Reply:
That did the trick, revision pushed to this PR. Thanks!